### DISCUSSION TITLE
Remove package published in error

### DISCUSSION THREAD

**Discussion by MrHinsh** [chocolatey/choco#3529] [Q&A] [answered]
My automated build system has a SNAFU and accidentally published a v17 package, but we are on v16 and this was not intended. The package is not yet approved, and I want to have it removed, deleted, never seen, never used... obliterated...

**Comment by gep13** [comment, marked as answer]
~~@MrHinsh what package are we talking about here?~~

Actually, never mind....

Seems like you are talking about this one: https://community.chocolatey.org/packages/vsts-sync-migrator/17.0.0-preview0001

One option would be to reject this package version, which would mean that this package version does not show up in any searches (either through the site or Chocolatey CLI), and also couldn't be installed directly.

**Reply by gep13** [reply]
Actually, since this is a trusted package, I am going to go ahead and pre-emptively reject this package version.

**Reply by MrHinsh** [reply]
Thanks dude! Much apriciated... 

We got a contribution and I missed that the contributor used "main" which triggers a preview release on our end. Adding logic into the build to reject in the future.

**Reply by gep13** [reply]
If/when you do start shipping v17, and you need that package version, we can un-reject it, to allow you to push the package version again.  Just let us know if/when you get there.

**Reply by MrHinsh** [reply]
I can always skip over it 😎

**Reply by mkevenaar** [reply]
`17.0.0-preview0001` is a pre-release version number, so version 17.0.0 could still be pushed.
